.github/workflows/ci.yml

```diff
@@ -1,8 +1,10 @@
 ---
 name: CI
 on:
-  - pull_request
-  - push
+  push:
+  pull_request:
```

Comment:
Now we still have the `pull_request` trigger. Can we reduce this to only having the `push`trigger?